src/main/webapp/common_util.js

```diff
@@ -13,16 +13,22 @@
 // limitations under the License.
 
 /*
- * Extracts video id from full url
- */ 
-function cleanseUrl(url) {
-  // Split web address from parameters, extract first parameter
-  // TODO: Add checks to make this work if video is not first parameter.
-  var videoId = url.split("?");
-  videoId = (videoId.length > 1) ? videoId[1].split("&")[0] : videoId[0];
-
-  // If param name present, remove it to isolate video Id.
-  videoId = videoId.replace("v=", "");
-  
+ * @param url: url of the video to be analyzed
+ * @returns: just the video id of that video
+ * Credit for RegEx method derived from top answer on: 
+ * https://stackoverflow.com/questions/28735459/how-to-validate-youtube-url-in-client-side-in-text-box
+ */	 
+function extractYouTubeUrl(url) {
+  var videoId = '';
```

Comment:
Can we use `let` here instead?

Here's a blogpost describing why this is useful:
https://evertpot.com/javascript-let-const/#:~:text=const%20only%20prevents%20re%2Dassigning,make%20the%20entire%20object%20immutable.&text=It's%20useful%20to%20use%20const,Stop%20using%20var%20.

Reply:
For some reason, this also causes an error.